src/scene.py

```diff
@@ -1,14 +1,14 @@
 import random
 from typing import Dict, List
-
 from src.common.point import Point
 from src.objects.agent import Agent
 from src.objects.floor import Floor
 from src.objects.portal import Portal
 from src.objects.object import Object
 from src.objects.npcs.enemies.training_dummy import TrainingDummy
-
-
+import json
```

Comment:
Group exports in accordance to PEP 8 Python standard. 
https://peps.python.org/pep-0008/#imports
```
1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.
```
Please use more blank lines, also.